receiver/sqlqueryreceiver/integration_test.go

```diff
@@ -34,6 +34,83 @@ import (
 	"go.opentelemetry.io/collector/pdata/pmetric"
 )
 
+func testMovieMetrics(t *testing.T, rm pmetric.ResourceMetrics, genreAttrKey string) {
```

Comment:
Not asking for this to be changed at all, but (since it looks like these functions got moved up) I tend to prefer putting functions below where they are called -- so a `main` function would be at the top, any functions it calls would be below it, any functions those call would be below them, etc.. This might be the opposite of the convention in Python 😄.

Reply:
I've reordered the functions as you requested. I'm used to having helper methods declared and implemented before they're called, so that was why I originally moved things around. I've reverted to be more consistent with the rest of the repository.